dev/com.ibm.ws.org.apache.cxf.cxf.rt.rs.mp.client.3.2/bnd.overrides

```diff
@@ -50,9 +50,14 @@ Import-Package: \
  com.ibm.websphere.ras,\
  com.ibm.wsspi.classloading,\
  com.ibm.ws.cdi.extension;resolution:=optional,\
+ com.ibm.ws.ffdc,\
  com.ibm.ws.jaxrs20.client.configuration
 # org.apache.cxf.jaxrs.provider.jsrjsonp;version="[3.1,4)",\
 
+DynamicImport-Package: \
+ javax.interceptor,\
+ com.ibm.ws.microprofile.faulttolerance.cdi
```

Comment:
Why are these needed?

Reply:
`javax.interceptor` is required for the `InvocationContext` - used in InterceptorInvoker and MPRestClientInvocationContextImpl.

`com.ibm.ws.microprofile.faulttolerance.cdi` is used to load the `FaultTolerance` annotation class.

These classes should only be required when using MP Rest Client _and_ MP Fault Tolerance, so that's why I used DynamicImport rather than normal Import.